middle_end/flambda2/to_cmm/to_cmm_static.ml

```diff
@@ -79,8 +79,11 @@ let static_boxed_number ~kind ~env ~symbol ~default ~emit ~transl ~structured v
       Cmmgen_state.add_structured_constant symbol_name structured_constant;
       env, res, updates
     | Var (v, dbg) ->
-      C.make_update env res dbg kind ~symbol:(C.symbol ~dbg symbol) v ~index:0
-        ~prev_updates:updates
+      let env, res, updates =
+        C.make_update env res dbg kind ~symbol:(C.symbol ~dbg symbol) v ~index:0
+          ~prev_updates:updates
+      in
+      env, res, updates
```

Comment:
Is this necessary ?

Reply:
it's not, I'll simplify this back to the original code, ^^